blobshare/blobshare.go

```diff
@@ -31,7 +31,7 @@ func zero(b []byte) {
 	}
 }
 
-func NewKey(password string) (*[32]byte, *[32]byte, error) {
+func NewKey(password []byte) (*[32]byte, *[32]byte, error) {
```

Comment:
Why did we change this to []byte?  Just trying to understand what the thought process was.

Reply:
This was done to allow blobshare to be used on sntrup shared keys, which are 32-byte sequences instead of strings.